# This file is used to configure the camera settings for the realsense cameras
shared_settings:
  color_profile: "640,480,30"
  depth_profile: "640,480,30"
  align_depth: true
  enable_depth: false
  enable_infra1: false
  enable_infra2: false
  initial_reset: false
  log_level: "warn"

# Camera settings
cameras:
  - name: "d415_top"
    serial_no: "319522065801"
    camera_name: "cam_top"
    enabled: true

  - name: "d415_right"
    serial_no: "241222061342"
    camera_name: "cam_right"
    enabled: true

  - name: "d415_left"
    serial_no: "241222061237"
    camera_name: "cam_left"
    enabled: false

  - name: "d415_first"
    serial_no: "241222063048"
    camera_name: "cam_first"
    enabled: false
    
  - name: "d415_far"
    serial_no: "327322062928"
    camera_name: "cam_far"
    enabled: false